app/controllers/discourse_ai/ai_bot/shared_ai_conversations_controller.rb

```diff
@@ -32,6 +32,10 @@ def destroy
       end
 
       def show
+        # we don't use an asset packager, so remember to bump this
+        # when CSS changes
+        @asset_version = 1
```

Comment:
Can you just use DiscourseAi::AiBot::SharedAiConversationsHelper::VERSION here?

Reply:
oh was left over code, removed it! :hugs: